src/components/HomePage/RecentBlogItem.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import BambooStraw from "../../images/BambooStraw";
+
+
+const RecentBlogItem = (title, description, image) => {
+
+  title = "Why buying bamboo straws ?!";
+  description =
+   "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quis netus gravida vitae at dui. Arcu ullamcorper eget lorem nulla sit. Elit varius felis praesent vel. Commodo quis nisi leo tristique in..."; 
+  image = <BambooStraw/>;
```

Comment:
why putting these data here? you can pass them as props when you call the component